packages/collector/test/tracing/sdk/app.js

```diff
@@ -166,41 +180,17 @@ app.post('/promise/create-intermediate', function createIntermediatePromise(req,
     });
 });
 
-app.post('/callback/create-overlapping-intermediates', async function createOverlappingIntermediatesCallback(req, res) {
```

Comment:
This test moved down but also changed/fixed. It did not actually test the functionality it was supposed to test (passing the span to completeIntermediateSpan). The test was still green, even when removing the extra `span2`/`span1` arguments provided to `completeIntermediateSpan` because it used the spans that were the currently active span at that point anyway.